### PR TITLE
Migrating dart:html to package:web

### PR DESCRIPTION
**Pull request by a-h-mzd**
This PR addresses the following issue:  #498

### CODE REVIEW

File: lib/fluttertoast_web.dart [hunk 1]
```diff
@@ -69,27 +69,29 @@ class FluttertoastWebPlugin {
...
+    final web.HTMLScriptElement script = web.HTMLScriptElement()
       ..async = true
       // ..defer = true
       ..src = jsUrl;
     loading.add(script.onLoad.first);
     tags.add(script);
-    html.querySelector('head')!.children.addAll(tags);
+    for (final web.HTMLElement tag in tags) {
+      web.document.querySelector('head')!.append(tag);
+    }
```
Comment (amrgetment) [inline lib/fluttertoast_web.dart:94]: querySelector has children, why for loop?

**Review by amrgetment — COMMENTED** [review]
You need to update index.html check my PR
https://github.com/miguelpruivo/flutter_file_picker/pull/1481/files

File: lib/fluttertoast_web.dart (hunk 1 above)
Comment (a-h-mzd) [reply lib/fluttertoast_web.dart:94]: Can you elaborate on how querySelector having children can help us get rid of the for loop? Maybe a code snippet?

**Comment by a-h-mzd** [comment]
> You need to update index.html check my PR https://github.com/miguelpruivo/flutter_file_picker/pull/1481/files

Actually, there is no need to change the example index.html file. The behavior you're referring to is going to change when flutter support for wasm lands on the stable channel. Also making the example project build with wasm is irrelevant to this PR.

File: lib/fluttertoast_web.dart (hunk 1 above)
Comment (amrgetment) [reply lib/fluttertoast_web.dart:94]: nevermind, you are right

**Review by amrgetment — APPROVED** [review]

**Comment by Mr-Pepe** [comment]
@ponnamkarthik Any chance to get this merged? :) This is a blocker for WASM builds.

**Comment by ponnamkarthik** [comment]
@Mr-Pepe pushed new version of the pulgin.

Thanks for the PR @a-h-mzd

**Comment by Mr-Pepe** [comment]
Wow, that was fast! Thank you :partying_face:

**Pull request merged**